js/events/index.d.ts

```diff
@@ -6,6 +10,50 @@ export interface EventType { }
  */
 export type TEvent = {} extends EventType ? dxEvent : EventType;
 
+/** @public */
+export interface ComponentEvent<T> {
+    component: T;
+    element: TElement;
+    model?: any;
+}
+
+/** @public */
+export interface ComponentNativeEvent<T> {
+    component: T;
+    element: TElement;
+    model?: any;
+    event?: TEvent;
+}
+
+/** @public */
+export interface ComponentDisposingEvent<T> {
+    component?: T;
+}
+
+/** @public */
+export interface ComponentInitializedEvent<T> {
+    component?: T;
+    element?: TElement;
+}
+
+/** @public */
+export interface ComponentOptionChangedEvent<T> {
+    component?: T;
+    name?: string;
+    fullName?: string;
+    value?: any;
+}
+
+export interface Cancelable {
+    cancel?: boolean;
+}
+
+export interface ItemInfo {
+    readonly itemData?: any;
+    itemElement: TElement;
```

Comment:
`ItemInfo` interface is used in most of `ItemXXX` events. Is `.itemElement` field is always writable?

/cc @DokaRus, @IgnatovDan

Reply:
"itemElement" is readonly on my sources.

I checked my sources and found this declaration for TabPanel:
    readonly itemElement?: TElement;

and this declaration for TreeView:
    readonly itemElement?: TElement;
    readonly itemIndex?: number;

and this declaration for Accordion:
    readonly itemElement: TElement;